### PR TITLE
Added alias_name and tags

### PR DESCRIPTION
**Pull request by tr0n-p46**
Added alias_name and tags to device response. 

**Note: Merge and push to pypi post SCAPI release

### CODE REVIEW

File: esper/controllers/device/device.py [hunk 1]
```diff
@@ -261,7 +297,10 @@ def set_active(self):
                     self.app.render(f'Device does not exist with name {device_name}\n')
                     return
                 response = search_response.results[0]
-                db.set_device({'id': response.id, 'name': response.device_name})
+                name = response.device_name
+                if response.alias_name and response.alias_name != '':
```
Comment (tarun91vas) [inline esper/controllers/device/device.py]: You are using multiple times. Can you make it a function ?

File: esper/controllers/device/device.py [hunk 2]
```diff
@@ -143,46 +159,63 @@ def list(self):
...
                 'model': "MODEL",
-                'state': "CURRENT STATE"
+                'state': "CURRENT STATE",
+                'tags': "TAGS"
             }
 
             for device in response.results:
                 current_state = DeviceState(device.status).name
+                name = device.device_name
+                if device.alias_name and not device.alias_name == '':
```
Comment (jerynmathew) [inline esper/controllers/device/device.py]: if `device.alias_name` should automatically check for empty strings as well.

File: esper/controllers/device/device.py (hunk 1 above)
Comment (tr0n-p46) [reply esper/controllers/device/device.py]: Done!

File: esper/controllers/device/device.py (hunk 2 above)
Comment (tr0n-p46) [reply esper/controllers/device/device.py]: Done!

**Comment by abdulMuneerM** [comment]
You have to spawn new branch from develop branch, and raise the PR into develop branch. Esper team will raised the PR into master branch.

**Pull request merged**